algorithm/calc/parameter.go

```diff
@@ -107,10 +107,10 @@ const (
 
 	// FALSE Dimension value of the empty geometry (-1).
 	// TRUE Dimension value of non-empty geometries (= {P, L, A}).
-	// DONTCARE Dimension value for any dimension (= {FALSE, TRUE}).
-	FALSE    = -1
-	TRUE     = -2
-	DONTCARE = -3
+	// DONOTCARE Dimension value for any dimension (= {FALSE, TRUE}).
+	FALSE     = -1
+	TRUE      = -2
+	DONOTCARE = -3
```

Comment:
"DONOT_CARE" is better ?

Reply:
Because I want to use camel case code style，const variable should be defined uniformly

Reply:
take and done

Reply:
OK